projects/cms-ui-demo/src/modules/app.component.ts

```diff
@@ -18,14 +19,18 @@ export class AppComponent implements OnInit {
 
   //#region Constructor
 
-  public constructor(protected titleService: Title) {
+  public constructor(
+    protected titleService: Title,
+    protected translateService: TranslateService
+  ) {
   }
 
   //#endregion
 
   //#region Methods
 
   public ngOnInit(): void {
+    this.translateService.use('en_US');
```

Comment:
Locale and localizatio file must be in hyphen naming convention. Such as: `en-US`